lib/Interactions/Internal/WebDriverKeyDownAction.php

```diff
@@ -2,9 +2,7 @@
 
 namespace Facebook\WebDriver\Interactions\Internal;
 
-use Facebook\WebDriver\WebDriverAction;
-
-class WebDriverKeyDownAction extends WebDriverSingleKeyAction implements WebDriverAction
```

Comment:
Why remove the Interface for these two files?

Reply:
WebDriverSingleKeyAction already implements WebDriverAction

Reply:
That should really be a separate pull request or, at the very least, a separate commit. It’s unrelated to the change in this PR.